spec/facility_spec.rb

```diff
@@ -24,3 +24,282 @@
     end
   end
 end
+
+require './lib/facility' = true
+
+require './lib/vehicle' = true
+
+facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
```

Comment:
Try to use a reference spec to guide you to write your tests. The interaction pattern will show you what kind of test cases to write, but check out the rest of this facility spec to follow the pattern created there with `it` blocks and `expect` statements. 

Again, we don't want you to copy and paste the interaction pattern,  but use  it to answer these questions:
- what attributes should the class set up objects with
- what methods should the objects have?

From there, you can write a test for every method and attribute